server/database/sql/insert_rows.sql

```diff
@@ -6,4 +6,4 @@ insert into role_lookup ( name, description ) values ( 'admin', 'administrator.
 /* FEATURE FLAGS */
 -- foreign key makes feature flags dependent on roles. they must be inserted after roles
 insert into feature_flags ( name, description, is_enabled ) values ( 'premiumDashboardSwitch', 'a switch that changes state from basic to premium dashboard. brings up upgrade popup for basic members', CAST(0 as BIT) );
-insert into feature_flags ( name, description, is_enabled, required_role ) values ( 'adminDashboardMenuItem', 'option in user menu that navigates to admin dashboard', CAST(0 as BIT), 3 );
+insert into feature_flags ( name, description, is_enabled, required_role ) values ( 'adminDashboardMenuItem', 'option in user menu that navigates to admin dashboard', CAST(1 as BIT), 3 );
```

Comment:
admin dashboard is enabled by default. in order to use it your role will have to be elevated to 3. I haven't added anything to the UI for controlling role yet, so if you want one of your accounts for testing develop elevated let me know and I'll show you how we can do that on the server